sass/application.scss

```diff
@@ -1,17 +1,3 @@
-/*
```

Comment:
yeah, if we start doing css right, it should prbly all be sass, so setting our sights on that

Reply:
I'd like to see each sass file be directly associated with (and imported by) exactly one react component. then all sass would be directly related to some js view code. If we can make any steps towards making that a reality in this PR let's do it. but we certainly won't get that job done as part of this, because it'll be big